src/Widgets/NavigationWidget/SubComponents/SearchBox.tsx

```diff
@@ -0,0 +1,43 @@
+import { connect, currentPage, navigateTo, Obj } from 'scrivito'
+import { useRef } from 'react'
+
+export const SearchBox = connect(function SearchBox({
+  searchResultsPage,
+}: {
+  searchResultsPage: Obj | null
+}) {
+  const inputRef = useRef<HTMLInputElement>(null)
+
+  if (!searchResultsPage) return null
+  const disabled = searchResultsPage.id() === currentPage()?.id()
+
+  return (
+    <form
+      role="search"
+      onSubmit={(event) => {
+        event.preventDefault()
+
+        const q = inputRef.current?.value
+        if (!q) return
+        inputRef.current.value = ''
+
+        navigateTo(searchResultsPage, { q })
+      }}
+    >
+      <div className="input-group" style={{ height: '100%' }}>
+        <input
+          className="form-control"
+          type="search"
+          placeholder="Search"
+          aria-label="Search"
+          ref={inputRef}
+          disabled={disabled}
+        />
+
+        <button type="submit" className="btn btn-primary" disabled={disabled}>
```

Comment:
You right, that the design HTML does not have a summit button. I haven't talk with Alex about it, since he was busy with many other projects. I was inspired by the button from https://uxbooth.com/articles/best-practices-for-search/.

Reply:
How about leaving it out while no UX expert is available?